flink/v2.0/flink/src/main/java/org/apache/iceberg/flink/maintenance/api/ExpireSnapshots.java

```diff
@@ -46,6 +46,7 @@ public static class Builder extends MaintenanceTaskBuilder<ExpireSnapshots.Build
     private Integer numSnapshots = null;
     private Integer planningWorkerPoolSize;
     private int deleteBatchSize = DELETE_BATCH_SIZE_DEFAULT;
+    private Boolean cleanExpiredMetadata = null;
```

Comment:
we may want to make this a primitive

Reply:
+1

```suggestion
    private boolean cleanExpiredMetadata;
```

Reply:
This means that if we change the default value, then we will have to change in 2 separate places (Spark and Flink).

Since we made the decision on the Spark side, we should behave the same here, but I see the logic with having an object instead of the primitive value here.